website/docs/reference/node-selection/defer.md

```diff
@@ -51,6 +45,14 @@ When using defer, you may be selecting from production datasets, development dat
 
 Deferral requires both `--defer` and `--state` to be set, either by passing flags explicitly or by setting environment variables (`DBT_DEFER_TO_STATE` and `DBT_ARTIFACT_STATE_PATH`). If you use dbt Cloud, read about [how to set up CI jobs](/docs/deploy/cloud-ci-job).
 
+<VersionBlock firstVersion="1.4">
```

Comment:
@matthewshaver I moved this further down, to allow for the full narrative flow above, and I switched it to using a `<VersionBlock>` instead of `<Changelog>` (legacy). Happy to hear disagreement!

Reply:
No disagreements! Thank you